From: Thomas Preud'homme <robotux@celest.fr>
Date: Sat, 7 Sep 2013 20:52:23 +0200
Subject: Define WCONTINUED and WIFCONTINUED if not defined

WCONTINUED and WIFCONTINUED are Linux-specific macros to the wait system
call. Thus, these macros might not be defined for other operating
systems such as Hurd. This patch define WCONTINUED and WIFCONTINUED in
such case so that the case of a child process receiving a SIGCONT is
simply not notified.

Origin: vendor
Bug: https://sourceforge.net/p/dspam/bug-tracker/169/
Last-Update: 2013-09-07
---
 src/external_lookup.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/external_lookup.c b/src/external_lookup.c
index 4f8e10e..8d0dcc9 100644
--- a/src/external_lookup.c
+++ b/src/external_lookup.c
@@ -51,6 +51,12 @@
 #   define BIND_TIMEOUT	10
 #endif
 
+/* Hurd */
+#ifndef WCONTINUED
+#define WCONTINUED 0
+#define WIFCONTINUED(status) 0
+#endif
+
 void 
 sig_alrm(int signum)
 {
